content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -105,6 +105,28 @@ in your **Hosted endpoints** and insert the following data into the pop-up form:
 When you click on **Add endpoint**, your Webhook is configured. You can then click on **reveal** to
 display your signing secret.
 
+Configuration tab
+-----------------
+
+.. _stripe/place_hold_on_card:
+
+Place a hold on a card
+~~~~~~~~~~~~~~~~~~~~~~
+
+With Stripe, you can enable the :ref:`manual capture <payment_acquirers/capture_amount>`. If
+enabled, the funds are reserved for 7 days on the customer's card, but not charged yet.
+
+.. warning::
+   Only some payments methods support separate authorization and capture.
+   
+   For example, card payments support separating these steps. The expiration window is 7 days.
+
+   With payment methods that don’t support this functionality, like iDEAL, you can’t separate
+   the authorization and capture steps.
```

Comment:
I wouldn't talk about other payment methods, I don't think it's useful since we're configuring stripe, I think the only important information here is the 7-day-window. However, I wouldn't write it like that, it looks like it's not important

Also I tried to rephrase it but I'm not convinced about what I came up with, is it the same as the Adyen 7-day-window? 
[https://www.odoo.com/documentation/master/applications/finance/payment_acquirers/adyen.html#place-a-hold-on-a-card](url)
```suggestion
    
   You have **7 days** to capture the amount. After this time, the manual capture expires.
```

Reply:
it suits me well :+1: